### DISCUSSION TITLE
Search on home page

### DISCUSSION THREAD

**Discussion by lephtHanded** [lllllllillllllillll/DweebUI#90] [Ideas]
Are there plans to implement a search for the container list?  (maybe a search bar next to the dark-mode toggle?)
- as we start typing the work 'dweeb,' it immediately shows only containers, first with D, then with DW  (kind of like **dashy** does?

Also tags?
Let's say I have a compose file that has several services in them (like 'infra' or 'public') could we set up a per-container environment variable that when we search, it displays the containers with that tag?

Just re-iterating how great the GUI is. 
- I was using portainer but didn't like how complicated it got - and the whole licensing thing
- I tried dockge - and it was more along the lines of what i wanted, but didn't have the central library of compose files that were deployed.
- that's why I ended up with dokemon - but it has some management issues I'm not excited about.

**Comment by lllllllillllllillll** [comment]
Yeah. There was one built into the navbar but I (temporarily) removed it.
I was going to keep it here. The top row of the navbar will eventually have shortcuts or toggles.
![search](https://github.com/lllllllillllllillll/DweebUI/assets/147879489/c8eaf59a-0a98-4926-ae95-8dc659b9e1ba)

**Reply by lllllllillllllillll** [reply]
> _I still need to implement a way to remove compose files once they're uploaded._

**Reply by lephtHanded** [reply]
> Yeah. There was one built into the navbar but I (temporarily) removed it. I was going to keep it here. The top row of the navbar will eventually have shortcuts or toggles....

This is a great place for the search.

**Reply by lephtHanded** [reply]
> > _I still need to implement a way to remove compose files once they're uploaded._

I was going to say the only thing missing was a place to paste your compose file in to a library.
(rather than uploading it first)
That library could be used to deploy to the different hosts.
